app/build.gradle.kts

```diff
@@ -33,12 +29,21 @@ android {
             )
         }
     }
+
+    buildFeatures {
+        viewBinding = true
+        compose = true
+    }
+
     compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_11
-        targetCompatibility = JavaVersion.VERSION_11
+        sourceCompatibility = JavaVersion.VERSION_17
```

Comment:
what does this do?
does it have an impact on users?

Reply:
These lines defines the available Java features available to the project, especially about the byte code. This was needed by some of the added libraries. This should not have impact on the users, as the compilation step should take care of desugaring the code for the older devices.